Portifolio/contact.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Contact | Luis Filipe</title>
+    <link rel="stylesheet" href="Style.css"/>
+</head>
+<body>
+    <header>
+        <div>
+            <a class="button1" href="home.html">About</a> 
```

Comment:
As a practice, give meaningful names for classes. In this case, you can name it `golden-button` and maybe `tomato-button` for the other class. Either way make sure by the name of the class, it would be ideal if readers can sense the purpose